Packs/AWS/Integrations/AWS/README.md

```diff
@@ -2562,3 +2562,83 @@ Describes resource discovery association with an Amazon VPC IPAM. An associated
 | AWS.EC2.IpamResourceDiscoveryAssociations.IpamResourceDiscoveryId | String | The resource discovery ID. |
 | AWS.EC2.IpamResourceDiscoveryAssociations.IpamRegion | String | The IPAM home region. |
 | AWS.EC2.IpamResourceDiscoveryAssociations.AccountId | string | The ID of the AWS account with which the EC2 instance is associated. This key is only present when the parameter "AWS organization accounts" is provided. |
+### aws-ec2-enable-imdsv2-quick-action
```

Comment:
The same in this README. Please update the title of the command to be the pretty name of the quick action, and leave the base command as it is now.